packages/common/src/content/types.ts

```diff
@@ -93,4 +92,7 @@ export interface IHubOtherContentStatus extends IHubBaseContentStatus {
   kind: "other";
   // TODO: expand in the future if needed
 }
+export interface IGetServiceStatusOptions extends IGetLayerOptions {
+  timeout?: number;
+}
```

Comment:
I think this should live next to `getServiceStatus` in `contentUtils.ts` and be exported from there